packages/@aws-cdk/aws-events-targets/lib/ecs-task.ts

```diff
@@ -95,6 +108,7 @@ export class EcsTask implements events.IRuleTarget {
   private readonly taskDefinition: ecs.TaskDefinition;
   private readonly taskCount: number;
   private readonly role: iam.IRole;
+  private readonly platformVersion: ecs.FargatePlatformVersion|undefined;
```

Comment:
```suggestion
  private readonly platformVersion?: ecs.FargatePlatformVersion;
```

rationale: you shouldn't need to use `| undefined` if the property is optional.